pre/main.ipynb

```diff
@@ -25,46 +25,94 @@
   },
```

Comment:
_Commented on notebook `pre/main.ipynb` Cell 2 Line 16_
```python
import datetime
from dockerfile_parse import DockerfileParser
from bspump.abc.generator import Generator
import docker
import docker.errors
import subprocess
import re
import bspump.kafka
```
What is kafka used for?

_Respond and view the context [here](https://gitnotebooks.com/bitswan-space/BitSwan/pull/13#comment-1141)._